extensions/amp-analytics/amp-analytics.md

```diff
@@ -164,6 +164,64 @@ Type attribute value: `quantcast`
 
 Adds support for Quantcast Measurement. More details for adding Quantcast Measurement can be found at [quantcast.com](https://www.quantcast.com/help/guides/)
 
+### SOASTA mPulse
+
+Type attribute value: `mpulse`
+
+Adds support for [SOASTA mPulse](https://www.soasta.com/mPulse). Configuration details can be found at [docs.soasta.com](http://docs.soasta.com/).
+
+Optional `extraUrlParams` parameters include `page_group`, `ab_test`, `custom_dimension.*` and `custom_metric.*` which are defined below. Note that since these optional parameters are included in the page output they may be cached by AMP proxy caches and served to multiple visitors.
```

Comment:
I'm concerned that adding this level of documentation detail here makes this page harder to parse. Would it be possible to fold this into documentation that lives at docs.soasta.com?